torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16763,19 +16752,6 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            supports_varargs=True,
            sample_inputs_func=sample_inputs_permute,
            reference_inputs_func=reference_inputs_permute),
-    OpInfo('permute_copy',
```

Comment:
what happened with all these tests??

Reply:
I temporarily removed `permute_copy`, `expand_copy` and `unbind_copy` because I was having difficulty getting them to work; my intention is to get everything else working and then add them one at a time, either in this commit or a second one right afterward, as well as `select_copy`, which also looks like a candidate.

Yes, I know it's a bit lame, but it looks like I did get the rest of it working, so I'll be on it immediately.